lib/web_ui/lib/src/engine/html/path/path_ref.dart

```diff
@@ -263,15 +263,12 @@ class PathRef {
         // The location delta of control point specifies corner radius.
         if (vector1_0x != 0.0) {
```

Comment:
`if (!SPath.nearlyEqual(pts[2], pts[0]))` ?

(Going by lines 256 and 291)

Reply:
For now just changing the asserts, but keeping the release functionality intact.